libpod/container.go

```diff
@@ -469,7 +472,40 @@ func (c *Container) Unpause() error {
 // Export exports a container's root filesystem as a tar archive
 // The archive will be saved as a file at the given path
 func (c *Container) Export(path string) error {
-	return ErrNotImplemented
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	if err := c.syncContainer(); err != nil {
+		return err
+	}
+
+	mountPoint := c.state.Mountpoint
+	if !c.state.Mounted {
+		mount, err := c.runtime.store.Mount(c.ID(), c.config.MountLabel)
+		if err != nil {
+			return errors.Wrapf(err, "error mounting container %q", c.ID())
+		}
+		mountPoint = mount
+		defer func() {
+			if err := c.runtime.store.Unmount(c.ID()); err != nil {
+				fmt.Printf("error unmounting container %q: %v", c.ID(), err)
```

Comment:
Probably should use logrus for this.  Not crazy about a libraryou outputing to STDOUT.  Do we/Should we pass in an IO device to be able to write these types of messages to?

Reply:
I think logrus should suffice for now. It might be nice to have a messages buffer in the future for things like conmon to write to

Reply:
fixed.
aren't these types of messages shown when the log-level flag is set?

Reply:
Well shown to different locations.  If you are running in a web service or some other non CLI tool they will end up in the syslog,  Rather then going out to a tty, which may or maynot exists.